internal/util/logging/handler_test.go

```diff
@@ -53,16 +53,29 @@ func TestHandler(t *testing.T) {
 			`{"function":"github.com/FerretDB/FerretDB/v2/internal/util/logging.TestHandler",` +
 			`"file":"logging/handler_test.go","line":34},"msg":"multi\nline\nmessage"` +
 			`,"g2":{"i":1,"g3":{"s":"a"},"name":"test.logger","g1":{"k1":42,"k2":7000000000},"k3":"s","k3":"dup"}}` + "\n",
+		"embeddable": `time=2024-05-31T09:26:42.000Z level=INFO msg="multi\nline\nmessage" ` +
+			`g2.i=1 g2.g3.s=a g2.name=test.logger g2.g1.k1=42 g2.g1.k2=7s g2.k3=s g2.k3=dup` + "\n",
 	} {
 		t.Run(base, func(t *testing.T) {
 			t.Parallel()
 
-			var buf bytes.Buffer
-			var h slog.Handler = NewHandler(&buf, &NewHandlerOpts{
-				Base:  base,
-				Level: slog.LevelInfo,
-			})
+			var (
+				buf bytes.Buffer
+				h   slog.Handler
+			)
```

Comment:
consistency with rest of our code :pray: 
```suggestion
			var buf bytes.Buffer
			var h   slog.Handler
```